tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": ["@railgun-reloaded/tsconfig/tsconfig"],
-  "include": ["src/**/*", "test/**/*.ts"],
-  "exclude": ["src/generated/**/*", "src/**/*.js", "test/**/*.js", "node_modules"]
+  "include": ["src/**/*.ts", "test/**/*.ts"],
+  "exclude": ["src/generated/**/*"]
```

Comment:
Is it correct to exclude these from TS? Do they not need to be built before publishing?